sys-devel/mlir/mlir-13.0.0.ebuild

```
@@ -0,0 +1,48 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_ECLASS=cmake
+
+inherit cmake-multilib llvm
+
+DESCRIPTION="Multi-Level IR Compiler Framework for LLVM"
+HOMEPAGE="https://mlir.llvm.org/"
+SRC_URI=" https://github.com/llvm/llvm-project/releases/download/llvmorg-13.0.0/llvm-13.0.0.src.tar.gz -> llvmorg-13.0.0.tar.gz"
```

You're hard coding version numbers here:
```suggestion
SRC_URI=" https://github.com/llvm/llvm-project/releases/download/llvmorg-${PV}/llvm-${PV}.src.tar.gz"
```

Why rename to 'llvmorg'? Actually, isn't this what `llvm.org.eclass` is for? (see the other LLVM ebuilds)